### ISSUE TITLE
Add Circular Spin Effect to Homepage Team Icons

### ISSUE THREAD

**Issue by JOwen-ster** [acmcsufoss/acmcsuf.com#1124]
### What's your idea?

it would look REALLY cool

**S P I N**
![image](https://github.com/user-attachments/assets/21e30bda-52dc-4c8a-b31e-bc9324e906e1)

**Comment by adriancancode** [comment]
U mean all the logos spinning around capy?  And yes it would look really dope

**Comment by adriancancode** [comment]
@JOwen-ster Imma create a branch fr this and see what I can do.

**Comment by 26samaahmed** [comment]
@adriancancode Can you put a hold on this? I want to assign this to new contributors and I'll find another issue you can solve. That logo also needs adjustments because we have to add the new node buds logo so let's leave this for now

**Comment by isliese** [comment]
https://github.com/user-attachments/assets/f92d194e-c5d5-4e0a-a682-bfb94a948360

I made the icons rotate and added a "Whoa!" when you hover capybara

**Comment by JOwen-ster** [comment]
I was thinking the icons would stay upright, they would just follow a circular path around the cappy and each individual icon would always be oriented how it was the my first image.

![Image](https://github.com/user-attachments/assets/f3a85205-4ecd-4c3b-90b8-62ea8d511c84)

**Comment by 26samaahmed** [comment]
Yes, we will need to separate the logos individually so they can rotate properly which I will be working on with @dianellasy !
